tests/bats/test_gpu_stress.bats

```diff
@@ -0,0 +1,76 @@
+# shellcheck disable=SC2148
+# shellcheck disable=SC2329
+
+: "${STRESS_PODS_N:=15}"
+: "${STRESS_LOOPS:=5}"
+: "${STRESS_DELAY:=30}"
+
+setup_file () {
+  load 'helpers.sh'
+  _common_setup
+  local _iargs=("--set" "logVerbosity=6")
+  iupgrade_wait "${TEST_CHART_REPO}" "${TEST_CHART_VERSION}" _iargs
+}
+
+setup() {
+  load 'helpers.sh'
+  _common_setup
+  log_objects
+}
+
+bats::on_failure() {
+  echo -e "\n\nFAILURE HOOK START"
+  log_objects
+  show_kubelet_plugin_error_logs
+  echo -e "FAILURE HOOK END\n\n"
+}
+
+# Expand pod YAML with indexes
+_generate_pods_manifest() {
+  local out="$1"
+  local template="tests/bats/specs/pods-shared-gpu.yaml"
+  : > "$out"
+  for i in $(seq 1 "${STRESS_PODS_N}"); do
+    sed "s/__INDEX__/${i}/g" "${template}" >> "$out"
+    echo "---" >> "$out"
+  done
+}
+
+@test "Stress: shared ResourceClaim across ${STRESS_PODS_N} pods x ${STRESS_LOOPS} loops" {
+  for loop in $(seq 1 "${STRESS_LOOPS}"); do
+    echo "=== Loop $loop/${STRESS_LOOPS} ==="
+
+    # Apply ResourceClaim
+    kubectl apply -f tests/bats/specs/rc-shared-gpu.yaml
+
+    # Generate and apply pods spec
+    manifest="${TMPDIR:-/tmp}/pods-shared-${loop}.yaml"
```

Comment:
can you use `${BATS_TEST_TMPDIR}` instead?

That's provided by the test suite, and sits properly in the hierarchy, unique to this test session.

It'a a directory that is specific to this test.

https://bats-core.readthedocs.io/en/stable/writing-tests.html

> $BATS_TEST_TMPDIR is a temporary directory unique for each test. Could be used to create files required only for specific tests.